src/app/components/Metadata/index.test.jsx

```diff
@@ -74,6 +74,28 @@ describe('Metadata', () => {
     '@BBCNews',
   );
 
+  metadataSnapshotTest(
+    'News article with title value',
```

Comment:
Because the `title` passed in for this test is `null` can you change the testDescription to reflect this please.